.env

```diff
@@ -0,0 +1,4 @@
+DB_HOST="localhost"
+DB_NAME="movies"
+DB_USERNAME="movies"
+DB_PASSWORD="password"
```

Comment:
We never want to commit .env since it has sensitive data in it, we only commit the sample version without data.